src/pages/settings/Profile/Contacts/ContactMethodDetailsPage.tsx

```diff
@@ -167,8 +167,11 @@ function ContactMethodDetailsPage({route}: ContactMethodDetailsPageProps) {
     }, [loginData?.validatedDate, loginData?.errorFields?.addedLogin]);
 
     useEffect(() => {
+        if (loginData?.validatedDate) {
+            return;
```

Comment:
We should also return early if loginData is not available to avoid issue https://github.com/Expensify/App/issues/58730